Astro 2/7 release notes (#1859)
astronomer/docs: astro/release-notes.md

**Latest CLI Version**: 1.10.0 ([Release notes](cli/release-notes.md))

## February 7, 2023

### Additional improvements 

> - The page for new Astro users who are not yet part of an Organization has been updated to provide more clear first steps.

Reviewer: ```suggestion
- The instructions on the <page name> for new Astro users who are not yet part of an Organization have been improved to make joining a Workspace easier.
```
Does this page have a name? For example, the log in page? What is the intent of the page? I've made some guesses, but I recommend a rewrite to include more information.

Author: Unfortunately no, but I think "welcome page" is a good replacement.

### Bug fixes

> - Removed broken date filtering functionality from the Lineage UI.

Reviewer 2: It's really a nit, I think "Removed date filtering functionality" sidesteps it :)

You can feel free to leave as-is, just a suggestion.

> - Removed broken date filtering functionality from the Lineage UI.

Author: ```suggestion
- Removed nonfunctioning date filtering functionality from the Lineage UI.
```

> - Fixed an issue where a you could unintentionally turn off DAG deploys on a Deployment by triggering an image deploy from older versions of the Astro CLI.

Author: ```suggestion
- Fixed an issue where a you could unintentionally turn off DAG deploys on a Deployment by triggering an image deploy from an older version of the Astro CLI.
```

> - Fixed an issue where a you could unintentionally turn off DAG deploys on a Deployment by triggering an image deploy from an older version of the Astro CLI.

Reviewer: ```suggestion
- Fixed an issue where triggering an image deploy from an older version of the Astro CLI could unintentionally turn off DAG deploys on a Deployment.
```
If you don't accept this edit, you'll need to change _an issue where a you_ to _an issue where you_. Remove the _a_.
